BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/ble/beacon/filter/RssiArmaFilter.java

```diff
@@ -0,0 +1,54 @@
+package com.nexenio.bleindoorpositioning.ble.beacon.filter;
+
+/**
+ * Created by leon on 20.12.17.
+ *
+ * This filter calculates its rssi on base of an auto regressive moving average (ARMA) It needs only
+ * the current value to do this; the general formula is  n(t) = n(t-1) - c * (n(t-1) - n(t)) where c
+ * is a coefficient, that denotes the smoothness - the lower the value, the smoother the average
+ * Note: a smoother average needs longer to "settle down"
+ * Note: For signals, that change rather frequently (say, 1Hz or faster) and tend to vary more a
+ * recommended value would be 0,1 (that means the actual value is changed by 10% of the difference
+ * between the actual measurement and the actual average) For signals at lower rates (10Hz)
+ * a value of 0.25 to 0.5 would be appropriate.
+ */
+
+public class RssiArmaFilter {
```

Comment:
Im not sure whether Filter is the correct terminology as it usually means you remove certain values. But what you rather do is an Aggregation or a Reduction. What about calling it `RssiArmaReducer` ?